device-types/Juniper/EX4100-48T.yaml

```diff
@@ -0,0 +1,135 @@
+---
+manufacturer: Juniper
+model: EX4100-48T
+slug: ex4100-48t
```

Comment:
Slugs should have the manufacturer appended to them. For this instance, `juniper-`